packages/nougat_cpu/tasks/extract_equations.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import requests
+import io
+
+from taskrunner import TaskRunnerInterface
+
+
+url = "http://localhost:8002/predict"
+
+
+def cleanup():
+    pass
+
+
+def main():
+    exitCode = 0
+    try:
+        taskrunner = TaskRunnerInterface(description="Embedding CLI")
+        taskrunner.on_cancellation(cleanup)
+
+        bs = taskrunner.read_input_bytes_with_timeout()
+
+        files = {"file": ("uploaded_file.pdf", io.BytesIO(bs), "application/pdf")}
+        response = requests.post(url, files=files)
+
+        response_json = response.json()
+
+        taskrunner.write_output_dict_with_timeout({"response": response_json})
+
+    except Exception as e:
+        sys.stderr.write(f"Error: {str(e)}\n")
+        sys.stderr.flush()
```

Comment:
Let's add stacktrace as well to make it easier to debug if any errors occur:

```
import traceback

... snipped ... 

sys.stderr.write(traceback.format_exc())
```

Reply:
Done